python/semantic_kernel/connectors/telemetry.py

```diff
@@ -26,19 +27,18 @@
 )
 
 
+APPLICATION_ID = f"{HTTP_USER_AGENT}/{version_info}"
+
+
 def prepend_semantic_kernel_to_user_agent(headers: dict[str, Any]):
     """Prepend "semantic-kernel" to the User-Agent in the headers.
 
     Args:
         headers: The existing headers dictionary.
 
     Returns:
-        The modified headers dictionary with "semantic-kernel" prepended to the User-Agent.
+        The modified headers dictionary with "semantic-kernel-python/{version}" prepended to the User-Agent.
     """
-    headers[USER_AGENT] = (
-        f"{HTTP_USER_AGENT}/{version_info} {headers[USER_AGENT]}"
-        if USER_AGENT in headers
-        else f"{HTTP_USER_AGENT}/{version_info}"
-    )
+    headers[USER_AGENT] = f"{APPLICATION_ID} {headers[USER_AGENT]}" if USER_AGENT in headers else APPLICATION_ID
```

Comment:
I get what `APPLICATION_ID` is, but per your comment in the `azure_ai_inference_base`, "application ID" feels like an internal concept to track usage. Our SK "user agent" is "semantic-kernel-python/version_info". Why does this need to be updated to be "APPLICATION_ID?"